inventory.yml.sample

```diff
@@ -45,6 +45,7 @@ all:
     # You will need to ensure alternatives are available for anything that will not be automatically set up
     setup_ntp_service: false
     setup_dns_service: false
+    setup_pxe_service: false
```

Comment:
Validate that if any vendors have `pxe` that the required fields e.g. `tftp_host` is present in the inventory validation playbooks.

Reply:
If you want to use your own PXE server it will fail